scripts/compare_deb_repo.py

```diff
@@ -23,15 +24,17 @@ def get_repo_packages(dist, release='nightly', arch='amd64'):
     return repo_packages
 
 
-def get_git_packages(dist):
+def get_git_packages(dist, release='nightly'):
     if dist == 'plugins':
-        package_folders = Path('plugins/').glob('*/debian')
+        package_changelogs = Path('plugins/').glob('**/changelog')
```

Comment:
Thinking out loud:  is it good to be more specific?
```suggestion
        package_changelogs = Path('plugins/').glob('*/debian/changelog')
```

Reply:
no, this was the point of the change, not all plugins have a debian subfolder :)